sdk/servicebus/perf-tests.yml

```diff
@@ -0,0 +1,44 @@
+Service: servicebus
+
+Project: sdk/servicebus/azure-servicebus
+
+PrimaryPackage: azure-eventhub
+
+PackageVersions:
+- azure-core: 1.26.1
+  azure-eventhub: 7.8.3
+- azure-core: source
+  azure-eventhub: source
+
+Tests:
+- Test: send-queue-message-batch
+  Class: SendQueueMessageBatchTest
+  Arguments:
+  - --event-size 1024 --batch-size 100 --parallel 64
```

Comment:
Is `event-size` the correct argument name or a carryover from the eventhubs perf configuration?

Reply:
No, Ive copied over the info from Anna's PR to fix it :)